allennlp/tests/commands/train_test.py

```diff
@@ -99,12 +99,9 @@ def test_train_model_distributed(self):
                 "train_data_path": SEQUENCE_TAGGING_DATA_PATH,
                 "validation_data_path": SEQUENCE_TAGGING_DATA_PATH,
                 "iterator": {"type": "basic", "batch_size": 2},
-                "trainer": {
-                    "num_epochs": 2,
-                    "optimizer": "adam",
-                    "distributed": True,
-                    "cuda_device": [0, 1],
-                },
+                "trainer": {"num_epochs": 2, "optimizer": "adam"},
+                "distributed": True,
```

Comment:
This flag is a bit pointless now - maybe we can just rely on distributed cuda_devices as the single flag?

Reply:
Less redundancy sounds good. Though see my point above about having a top-level `distributed: {...` stanza.